src/libraries/System.Numerics.Vectors/tests/Generation/Test/Matrix3x2_DoubleTests.cs

```diff
@@ -0,0 +1 @@
+using Matrix3x2Double = System.Numerics.Matrix3x2<System.Double>;
```

Comment:
Would be good to not use `using` like this

Reply:
The problem is it becomes much hard to generate tests without it, as you can't simply insert the type name into the test name because `<>` are not legal chars in method name ofc

Reply:
Sure, so use two different replacements. One is the name with <> and one is the name without